plugin.video.rsiplaytv/resources/settings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
can you use the new format for settings? There's an addon in the forum to autoconvert them for you